### PR TITLE
Deploy Grafana

### PR DESCRIPTION
**Pull request by upodroid**
Busy PR but:
- Grafana has been deployed and is reading metrics from Managed Prometheus https://grafana.knative.dev
- I need to create the prom rules and deploy them. (WIP)
- Created backing infra in GCP
- Gateway API is in beta but it is much better than ingress.

/cc @kvmware

### CODE REVIEW

File: infra/gcp/boskos/main.tf [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  required_providers {
-    google = {
-      version = "4.15"
-    }
-    google-beta = {
-      version = "4.15"
-    }
-  }
-}
+# terraform {
```
Comment (kvmware) [inline infra/gcp/boskos/main.tf:3]: Lots of commenting out through out this PR, is that necessary? Can't we just remove and if we need it back we can revert?

**Comment by kvmware** [comment]
Other than the comment i left LGTM

File: infra/gcp/boskos/main.tf (hunk 1 above)
Comment (upodroid) [reply infra/gcp/boskos/main.tf:3]: Fixed

Comment (upodroid) [reply infra/gcp/boskos/main.tf:3]: Oops forgot to push, afk

Comment (upodroid) [reply infra/gcp/boskos/main.tf]: This is ready now.

**Pull request merged**